index.js

```diff
@@ -31,6 +33,41 @@ async function run() {
     }
     containerDef.image = imageURI;
 
+    // Insert environment variable, if present
+    if (environmentVariables) {
+
+      // If environment array is missing, create it
```

Comment:
nit: I think we can remove the comments in the if-block, the code is pretty clear :)